### ISSUE TITLE
IdlePerloader breaks ngc

### ISSUE THREAD

**Issue by flauc** [angular/universal-starter#285]
I think adding the `idlePreloader` breakes ngc. This is the error i get: 

```
ERROR in [default] C:\Users\Detectives\Desktop\Projects\universal-starter\src\browser.module.ngfactory.ts:364:68 
    Supplied parameters do not match any signature of call target.
```

That's this line of the `.ngfactory`:

```ts
  get _IdlePreload_60():import12.IdlePreload {
    if ((this.__IdlePreload_60 == null)) { (this.__IdlePreload_60 = new import12.IdlePreload()); }
    return this.__IdlePreload_60;
  }
```

**Comment by gdi2290** [comment]
sorry about that. it's fixed in 1.0.4

**Comment by flauc** [comment]
Thanks for the quick response 👍
